### DISCUSSION TITLE
ANOOPSYCHE ‎SWA1

### DISCUSSION THREAD

**Discussion by ligi** [arendst/Tasmota#14008] [Q&A] [answered]
I have this device: https://www.amazon.de/-/en/Anoopsyche-European-Intelligent-Control-Function/dp/B0757MMM6G
Could flash tasmota without a problem - but I fail to be able to switch the relais.
I tried all configurations from anoopsyche templates I found thinking maybe there is a similar device that shares configuration - unfortunately it does not work. Also tried to make all GPIOs the relais (must be a relais in there as it is clicking) and then tried to toggle - but hear nothing. 
Is there any way to proceed from here or must I try to revert to the old firmware? Would really love to use this one as I prefer a open solution to a closed cloud solution.

**Comment by sfromis** [comment]
Are you trying the all-relay config while still on the USB/serial adapter for flashing? Relays may not click without mains power. Instead of listening for clicks, more or less audible, better have something like a lamp to give a clear indication of power or not.

**Comment by ligi** [comment]
Thanks for asking - but I am connected to mains - I updated the firmware wireless

**Comment by rapi3** [comment, marked as answer]
try this template it must work or your device it is faulty.
![Untitled](https://user-images.githubusercontent.com/18531150/145722744-a5b8471e-1b5e-4ced-9996-06e0eca64101.png)
